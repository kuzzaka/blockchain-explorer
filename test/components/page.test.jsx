import React from 'react';
import { shallow } from 'enzyme';
import Page from '../../components/page';

test('Page snapshot', () => {
  const wrapper = shallow(<Page title="My Title" />);
  expect(wrapper).toMatchSnapshot();
});
