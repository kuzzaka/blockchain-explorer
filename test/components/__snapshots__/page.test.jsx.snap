// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot 1`] = `
<main>
  <Header
    title="Blockchain Explorer"
  />
  <Connect(Summary) />
  <Connect(TransactionList) />
</main>
`;
